src/components/PressableOpacity.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement, ReactNode, useState } from 'react'
```

Comment:
Didn't realize there is no touch effects anymore in Pressable compared to TouchableOpacity. Do you think this PR makes still sense then? Or should we rather just keep TouchableOpacity wherever we want that effect?

Reply:
https://blog.logrocket.com/react-native-touchable-vs-pressable-components/
Well i think we can more standardize our components and even implement fancy animations if needed. Stuff like background-color change or opacity change can be implemented easily via styled components and is better readable and customizable for us. Thats my personal opinion

Reply:
Sounds good, lets keep this!